widgets/editor/editor.php

```diff
@@ -115,6 +117,10 @@ function get_style_name($instance) {
 		// We're not using a style
 		return false;
 	}
+
+	function enqueue_tinymce() {
+		wp_enqueue_style( 'editor-buttons' );
```

Comment:
I did a quick check and it looks like this isn't necessary at all anymore.

Reply:
That's odd. I initially attempted removing just [this code](https://github.com/siteorigin/so-widgets-bundle/pull/765/commits/d80902c3304db783330ff9dd997247b44774904c#diff-42c27934512039924cc8e53dcad763cdL249) from base/base.php without adding the highlighted code but I received SiteOrigin Page Builder generic error message stating that the widget styles and/or JavaScript weren't able to loaded (I can't remember the exact error message). I can't seem to replicate that error again, however.

Reply:
It might be okay to revert the editor.php changes, but I feel it's safer to include it anyway. If you're not worried about it then I'll revert those changes so the only adjustment is to base.php.

Reply:
This code was first introduce in 2015 before the core Text Widget was an instance of TinyMCE editor and it seems the editor button styles are now being enqueued properly by WordPress since that change. I prefer not to have redundant code around, so I think we should remove it and in the unlikely event that it was somehow useful, we can add it back with a decent comment stating the exact purpose.